app/konnect/analytics/index.md

```diff
@@ -15,62 +15,66 @@ historical data for the following lengths of time:
 
-![service graph](/assets/images/docs/konnect/konnect-vitals-service-versions.png)
+![service graph](/assets/images/docs/konnect/konnect-vitals-service-versions.png){:.image-border}
 
 > _**Figure 1:** Graph showing throughput for a service with interval filter options._
 
-For greater insights into your service usage, access the dedicated {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/analytics) page <span class="badge plus"></span>.
+You can also [export this historical data in CSV format](/konnect/analytics/services-and-routes/) for any individual service, service version, or route.
+
+
+## Summary dashboard and custom reports
+{:.badge .plus}
+
+For greater insights into your service usage, access the dedicated {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/analytics) page.
+
+From {% konnect_icon analytics %} Analytics, you can view dashboards, access historical data for a range greater than 30 days, and customize the entities in a report:
+* View the Analytics summary dashboard to track traffic, errors by error code, and latency across all services in your organization.
+* Create a [custom report](/konnect/analytics/use-cases/) for any number of services, service versions, routes, applications, or status codes.
+You can export any custom report as a CSV file.
+
+The summary dashboard provides metrics for services cataloged by Service Hub within a selected time interval for the following categories:
+
+* **Traffic**: Total number of HTTP requests.
+* **Errors**: Percentage of failed HTTP requests.
+* **Latency**: Request latency at P50, P95, and P99 percentiles.
+
+These categories measure trends by comparing metrics across fixed comparable time intervals and plotting the data points. For example, hour-over-hour, day-over-day, week-over-week, and month-over-month.
 
-From Analytics, you can view dashboards, access historical data for a range greater than 30 days, and customize the entities in a report:
-* View the [Analytics dashboard](/konnect/analytics/summary-dashboard) to track traffic, errors by error code, and latency across all services in your organization.
-* [Export historical data in CSV format](/konnect/analytics/services-and-routes/) for any individual service, service version, or route.
-* [Create a custom report](/konnect/analytics/generate-reports/) for any number of services, routes, or applications, filtered by time frame and grouped by metric.
+You can view a graph for each category by clicking **Traffic**, **Errors**, or **Latency**, and switching between the views.
 
-![traffic analytics graph](/assets/images/docs/konnect/konnect-vitals-traffic.png)
-> _**Figure 2:** Graph showing successful and failed requests over the past three hours._
+* **Traffic**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval.
+    * Successful requests contain all requests that returned a **2xx-3xx** status code.
```

Comment:
Could we update this to display **1xx-3xx**? Data team made some recent changes to include 1xx status codes in Successful requests.